.circleci/config.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - restore_cache:
           keys:
             - cache-pip
-      - run: pip install --user numpy scipy matplotlib seaborn sphinx_rtd_theme pillow sphinx pytest vtk traits traitsui pyface pyqt5 mayavi memory_profiler
+      - run: pip install --user -r doc/doc-requirements.txt
```

Comment:
note - this PR also splits out the docs requirements into their own file to make things clearer for would-be developers or users.